src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -58,9 +62,32 @@ public static Client init(Context androidContext, String apiKey, boolean enableE
      */
     public static Client init(Context androidContext, Configuration config) {
         client = new Client(androidContext, config);
+        configureClientObservers();
         return client;
     }
 
+    private static void configureClientObservers() {
+
+        // Ensure that the bugsnag observer is registered
+        // Should only happen if the NDK library is present
+        try {
+            String className = "com.bugsnag.android.ndk.BugsnagObserver";
+            Class c = Class.forName(className);
+            Observer o = (Observer)c.newInstance();
+            client.addObserver(o);
+        } catch (ClassNotFoundException e) {
+            // ignore this one, will happen if the NDK plugin is not present
+            Logger.warn("Failed to find NDK observer");
```

Comment:
I think the log level here should be lower, since it would be confusing for non-NDK consumers to see this.